py/picca/pk1d/postproc_pk1d.py

```diff
@@ -205,229 +200,791 @@ def compute_mean_pk1d(p1d_table,
     One row per z bin; column values z_min/max, k_min/max, N_chunks
     """
     # Initializing stats we want to compute on data
-    stats_array = ['mean', 'error', 'min', 'max']
+    stats_array = ["mean", "error", "min", "max"]
     if not nomedians:
-        stats_array += ['median']
+        stats_array += ["median"]
 
     p1d_table_cols = p1d_table.colnames
+    p1d_table_cols.remove("forest_id")
+    if "sub_forest_id" in p1d_table_cols:
+        p1d_table_cols.remove("sub_forest_id")
 
     # Convert data into velocity units
     if velunits:
-        conversion_factor = (ABSORBER_IGM["LYA"] *
-                             (1. + p1d_table['forest_z'])) / SPEED_LIGHT
-        p1d_table['k'] *= conversion_factor
+        conversion_factor = (
+            ABSORBER_IGM["LYA"] * (1.0 + p1d_table["forest_z"])
+        ) / SPEED_LIGHT
+        p1d_table["k"] *= conversion_factor
         for col in p1d_table_cols:
-            if 'Pk' in col:
+            if "Pk" in col:
                 p1d_table[col] /= conversion_factor
 
     # Initialize mean_p1d_table of len = (nzbins * nkbins) corresponding to hdu[1] in final ouput
     mean_p1d_table = Table()
     nbins_z, nbins_k = len(zbin_edges) - 1, len(kbin_edges) - 1
-    mean_p1d_table['zbin'] = np.zeros(nbins_z * nbins_k)
-    mean_p1d_table['index_zbin'] = np.zeros(nbins_z * nbins_k, dtype=int)
-    mean_p1d_table['N'] = np.zeros(nbins_z * nbins_k, dtype='int64')
+    mean_p1d_table["zbin"] = np.zeros(nbins_z * nbins_k)
+    mean_p1d_table["index_zbin"] = np.zeros(nbins_z * nbins_k, dtype=int)
+    mean_p1d_table["N"] = np.zeros(nbins_z * nbins_k, dtype=int)
     for col in p1d_table_cols:
         for stats in stats_array:
             mean_p1d_table[stats + col] = np.ones(nbins_z * nbins_k) * np.nan
 
     # Initialize metadata_table of len = nbins_z corresponding to hdu[2] in final output
     metadata_table = Table()
-    metadata_table['z_min'] = zbin_edges[:-1]
-    metadata_table['z_max'] = zbin_edges[1:]
-    metadata_table['k_min'] = kbin_edges[0] * np.ones(nbins_z)
-    metadata_table['k_max'] = kbin_edges[-1] * np.ones(nbins_z)
+    metadata_table["z_min"] = zbin_edges[:-1]
+    metadata_table["z_max"] = zbin_edges[1:]
+    metadata_table["k_min"] = kbin_edges[0] * np.ones(nbins_z)
+    metadata_table["k_max"] = kbin_edges[-1] * np.ones(nbins_z)
+
+    if (compute_covariance) | (compute_bootstrap):
```

Comment:
use `or` unless you are doing a bitwise OR (it's not the case here, is it?) Also you can drop the unnecessary parenthesis

Reply:
I fixed this one